README.md

```diff
@@ -1,233 +1,12 @@
 venv-update
---------
+===========
 Quickly and exactly synchronize a virtualenv with a requirements.txt.
 
 [![PyPI version](https://badge.fury.io/py/venv-update.svg)](https://pypi.python.org/pypi/venv-update)
-[![Build Status](https://travis-ci.org/Yelp/venv-update.svg?branch=master)](https://travis-ci.org/Yelp/venv-update)
+[![Build Status (circle)](https://circleci.com/gh/Yelp/pip-faster)](https://circleci.com/gh/Yelp/venv-update.svg?style=shield)
+[![Build Status (travis)](https://travis-ci.org/Yelp/venv-update.svg?branch=master)](https://travis-ci.org/Yelp/venv-update)
 [![Coverage Status](https://codecov.io/github/Yelp/venv-update/coverage.svg?branch=master)](https://codecov.io/github/Yelp/venv-update?branch=master)
+[![Documentation](https://readthedocs.org/projects/venv-update/badge/)(http://venv-update.readthedocs.org/en/latest/)
```

Comment:
It's the third line of this file, six lines up from here.
It's one of the only things you'll see if you look at the rendered markdown.